pkgs/cupertino_http/src/CUPHTTPClientDelegate.m

```diff
@@ -42,7 +42,7 @@ @implementation CUPHTTPClientDelegate {
 - (instancetype)init {
   self = [super init];
   if (self != nil) {
-    taskConfigurations = [NSMapTable strongToStrongObjectsMapTable];
+   taskConfigurations = [[NSMapTable strongToStrongObjectsMapTable] retain];
```

Comment:
Why are you building this code without ARC?

Reply:
Turning off ARC seems like a very large and dangerous hammer to solve that problem. Why not just have a collection where you keep objects whose lifetime you want to guarantee? That's the standard way to manage lifetimes in modern ObjC code.

Reply:
That's a great idea! I made a note to do it that way.